README.md

```diff
@@ -1945,11 +1956,14 @@ Some points on negotiating compensation:
   agree what those could look like *if* you demonstrate outstanding performance and the
   company has money.
 - If you’re moving from an established company to a startup, you may be asked to take a
-  salary cut. This is reasonable, but it’s wise to discuss explicitly how much the cut is,
-  and when your salary will be renegotiated.
-  For example, you might take 25% below your previous salary, but there can be an agreement
-  that this will be corrected if your performance is strong and the company gets funding.
-- 🔹 Always negotiate non-compensation aspects before agreeing to an offer.
+  [salary cut](http://fortune.com/2016/06/13/new-job-pay-cut-leadership/). This is reasonable,
```

Comment:
I had a hard time finding a good link here. Again, thick mire. I replaced link with Quora thread. I found this link (that is authoritative and recent) that would be so interesting and maybe have different results if the data weren't collected from blinkin' Denmark. Unusable? https://insights.som.yale.edu/insights/do-startups-pay-less

Reply:
I don't think unusable! This actually touches on some super helpful points, like, equity pay is better than unemployment, the jobs themselves are great. And it also states clearly the differences b/n US compensation and Danish compensation--namely, the Danish state actually provides for its citizens, so you're not talking about benefits reduction or anything like that. Now, the main study isn't actually out yet, but with a quick caveat this would be a fine link (just hopefully not the only one).